### DISCUSSION TITLE
Sharing my ESP Presense in a box

### DISCUSSION THREAD

**Discussion by agidi** [ESPresense/ESPresense#481] [General]
Just wanting to say thanks to the authors for such a cool project. Keep up the great work

I shared my implementation after some testing.
https://www.reddit.com/r/homeassistant/comments/ux1un1/esp_presence_in_a_box/?utm_source=share&utm_medium=web2x&context=3

I also setup a crude but simple wiring diagram to help others.
ESP32 with ESP presense  with PIR and Microwave radar
https://cloud2.neurotechmexico.com/index.php/s/ewSEdB46jyPxwwj

**Comment by erwinSmith757** [comment]
this is amazing, the code for this? this would be the ultimate sensor :)

**Comment by DTTerastar** [comment]
Yes! Very cool!  I assume there isn't code, just picking those 2 pins in the gui?
